src/app/ubs/ubs/components/ubs-personal-information/ubs-personal-information.component.html

```diff
@@ -138,6 +139,16 @@ <h2 class="address-title">{{ 'personal-info.info-address' | translate }}</h2>
       >
         {{ 'personal-info.info-add-address' | translate }}
```

Comment:
you can write here just `{{ (addresses.length === 0 ? 'personal-info.info-add-first-address' : 'personal-info.info-add-address') | translate }}`. You don't need to create a separate service for addresses (if only because we already have a service for addresses -`location.service.ts`)

Reply:
It's funny after what I did, but you're right. It was worth trying to learn to do services. Thank you!

Reply:
Your solution is not bad, but if we can make it in one line of code, why not))